src/OpenSSL/py.typed

```diff
@@ -0,0 +1 @@
+Marker
```

Comment:
This file should be empty

Reply:
Huh. For some reason I've been putting this same content all over my projects for years, not sure where I got it from?! I'll clear it.